### DISCUSSION TITLE
Missing files in package?

### DISCUSSION THREAD

**Discussion by Kraven3030** [themesberg/flowbite#402] [General]
When I download the packages for flowbite and start to implement a carousel in my react app, I receive an error message stating this...
"WARNING in ./node_modules/flowbite/lib/esm/components/tooltip/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from "
.../node_modules/flowbite/src/components/tooltip/index.ts' 
file: Error: ENOENT: no such file or directory, open 

Would anyone be able to help with this?

**Comment by zoltanszogyenyi** [comment]
Hello @Kraven3030,

We need a bit more information about your setup and project - perhaps you missed something?

Here's a guide that for Vue that could help with React as well: https://flowbite.com/docs/getting-started/vue/

Also if you use React you might want to take a look at the Flowbite React library: https://github.com/themesberg/flowbite-react

Cheers,
Zoltan
